py3status/modules/volume_status.py

```diff
@@ -200,17 +200,17 @@ def setup(self, parent):
         self.device_type_pl = self.device_type + "s"
         self.device_type_cap = self.device_type[0].upper() + self.device_type[1:]
 
-        self.reinit_device = self.device is None
-        if self.device is None:
+        self.use_default_device = self.device is None
+        if self.use_default_device:
             self.device = self.get_default_device()
 
         self.max_volume = parent.max_volume
         self.update_device()
 
     def update_device(self):
         self.re_volume = re.compile(
-            r"{} \#{}.*?State: (\w+).*?Mute: (\w{{2,3}}).*?Volume:.*?(\d{{1,3}})\%".format(
-                self.device_type_cap, self.device
+            r"{} (?:\#{}|.*?Name: {}).*?Mute: (\w{{2,3}}).*?Volume:.*?(\d{{1,3}})\%".format(
+                self.device_type_cap, self.device, self.device
```

Comment:
False positives can be happening here with percent displaying (but not allowing usres to scroll). I don't know regex... I take it that it will try to capture `#0` or `Name: 0`... annd...  `#example` or `Name: example`?